.github/workflows/main.yml

```
@@ -14,15 +17,16 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    name: Build
```

```suggestion
    name: build
```

I'm not sure about the name here